README.md

```diff
@@ -135,6 +131,33 @@ Yes, it's possible. Edit `prestart` and `start` scripts in package.json file acc
 "start": "contentful-extension-scripts start --port 8000",
 ```
 
+### I use development mode but my extension is not showing up. What's wrong?
+
+In development mode, extension is served from `http://localhost:1234`, but `app.contentful.com` is loaded via `https` protocol which leads to `mixed content` issue. While doing development you need to click **Load unsafe scripts** (Chrome) to enable serving extension from `http://localhost`.
```

Comment:
I'd explain in detail what's the "issue": say something like "which leads the browser to prevent mixed content" and link to https://developers.google.com/web/fundamentals/security/prevent-mixed-content/fixing-mixed-content